common/lib/dependabot/clients/bitbucket.rb

```diff
@@ -52,6 +68,92 @@ def fetch_file_contents(repo, commit, path)
         response.body
       end
 
+      def commits(repo, branch_name = nil)
+        commits_path = "#{repo}/commits/" + branch_name.to_s
+        commits_path += "?pagelen=100"
```

Comment:
This only returns the last 100 commits; should this go paginate through all commits instead?

Reply:
I suspect other clients do, but 100 commit seems like it might be "enough".
If it's working for you, works for me...

Reply:
Meh. I went ahead and implemented pagination for both commits and pull requests so someone else doesn't have to do it later.